### PR TITLE
Fix: Container - Widget width control is broken when Container experiment is active [ED-6565]

### PR DESCRIPTION
**Pull request by StyleShit**

### CODE REVIEW

File: includes/widgets/common.php [hunk 1]
```diff
@@ -258,15 +258,12 @@ private function register_layout_section() {
...
-						'condition' => [
-							'_element_width_{{DEVICE}}' => [ 'initial' ],
-						],
-					] ),
+				'condition' => [ '_element_width' => 'initial' ],
+				'device_args' => $this->get_responsive_device_args( [
+					'condition' => [
+						'_element_width_{{DEVICE}}' => [ 'initial' ],
+					],
+				] ),
```
Comment (StyleShit) [inline includes/widgets/common.php:266]: That's reverted to the original behavior:

https://github.com/elementor/elementor/blob/master/includes/widgets/common.php#L1274-L1281

File: assets/dev/js/editor/elements/views/behaviors/widget-resizeable.js [hunk 2]
```diff
@@ -97,27 +151,21 @@ export default class extends Marionette.Behavior {
...
-			settingToChange._flex_size = 'none';
-
-			if ( isContainer ) {
-				settingToChange.content_width = 'full';
-			}
-		} else {
-			settingToChange[ '_element_width' + deviceSuffix ] = 'initial';
-		}
+		const elementWidthSetting = this.getDeviceSetting( '_element_width' ),
+			widthSetting = this.getDeviceSetting( this.getWidthKey() );
```
Comment (matipojo) [inline assets/dev/js/editor/elements/views/behaviors/widget-resizeable.js:155]:
```suggestion
		const elementWidthSettingKey = this.getDeviceSetting( '_element_width' ),
			widthSettingKey = this.getDeviceSetting( this.getWidthKey() );
```

File: assets/dev/js/editor/elements/views/behaviors/widget-resizeable.js [hunk 3]
```diff
@@ -97,27 +151,21 @@ export default class extends Marionette.Behavior {
...
+
+		const { unit } = this.getSetting( widthSetting ),
```
Comment (matipojo) [inline assets/dev/js/editor/elements/views/behaviors/widget-resizeable.js:157]:
```suggestion
		const { unit } = this.getSetting( widthSettingKey ),
```

File: assets/dev/js/editor/elements/views/behaviors/widget-resizeable.js [hunk 4]
```diff
@@ -97,27 +151,21 @@ export default class extends Marionette.Behavior {
...
+			width = elementor.helpers.elementSizeToUnit( this.$el, ui.size.width, unit );
 
-		settingToChange[ widthKey + deviceSuffix ] = { unit, size: width };
+		const settingToChange = {
+			...( this.isContainerActive() ? { _flex_size: 'none' } : {} ),
+			...( this.isContainer() ? { content_width: 'full' } : {} ),
+			[ elementWidthSetting ]: 'initial',
+			[ widthSetting ]: {
```
Comment (matipojo) [inline assets/dev/js/editor/elements/views/behaviors/widget-resizeable.js:164]:
```suggestion
			[ elementWidthSettingKey ]: 'initial',
			[ widthSettingKey ]: {
```

**Review by matipojo — CHANGES_REQUESTED** [review]

**Review by matipojo — CHANGES_REQUESTED** [review]
Please resolve the conflicts

**Pull request merged**